combat-trainer.lic

```diff
@@ -138,7 +138,19 @@ class SetupProcess
       echo('skipping summoned weapons because no moonblade available') if $debug_mode_ct
       weapon_training = weapon_training.reject { |skill, _| game_state.summoned_info(skill) }
     end
-    new_weapon_skill = weapon_training.min_by { |skill, _| [DRSkill.getxp(skill), DRSkill.getrank(skill)] }.first
```

Comment:
I'm pretty sure there's a way to use `.sort` here to make this code a lot simpler. Something like "sort by the index of the weapon skill in `priority_weapons`" (and account for skills that aren't in the priority list), lower is better.